congress/static/congress/angular/app.js

```diff
@@ -211,14 +211,38 @@
 		// Boolean used for see if data have loaded
 		$scope.hasData = false
 
+		$scope.currentYears = []
+		let currentYear = new Date().getFullYear()
+		const counterYear = 2016
+		while(counterYear <= currentYear)
```

Comment:
This block without braces can be a bit confusing

Reply:
But without braces means that only execute one line. I think is not confusing at all

Reply:
It's just a matter-of-taste, I'm used to writing the braces for every block.
But feel free to leave it as it is.

Reply:
Done